fichero.py

```diff
@@ -1 +1,2 @@
 print(3+4)
+button("red")

```

Comment:
El botón debía ser rojo oscuro. 
```suggestion
button("dark_red")
```